apps/journeys-e2e/src/integration/firstStep.spec.ts

```diff
@@ -0,0 +1,25 @@
+describe('journeys', () => {
```

Comment:
can this file get a better name. Like [journeyId].spec.ts to match the nextjs page thats being tested.

```suggestion
describe('JourneyPage', () => {
```